package.json

```diff
@@ -2,7 +2,9 @@
   "name": "synctos",
   "version": "1.10.0",
   "main": "src/index.js",
-  "dependencies": {},
+  "dependencies": {
+    "joi": "^13.1.1"
```

Comment:
As I mentioned in the pull request description, I will remove this dependency in a subsequent pull request that will embed it as a static library in the `lib` directory instead.